packages/ceramic-doctype-tile/src/tile-doctype.ts

```diff
@@ -73,9 +73,19 @@ export class TileDoctype extends Doctype {
             throw new Error('No DID authenticated')
         }
 
+        // TODO: stop checking metadata for 'isUnique' once all dependent libraries are using DocParams to specify it
```

Comment:
While testing I got burned once again by the circular dependency with identity-wallet.  Since this patch changes the way isUnique is passed as an argument to document creation, [this line](https://github.com/3box/identity-wallet-js/blob/443f9b68f514951c6063f4f991d15cd51e3c645e/src/keyring.ts#L193) in identity wallet needs to be updated to use the new format.  So to work around that, I added extra handling here so that we continue to check the metadata temporarily as a fallback to passing isUnique in docParams.  Once identity wallet is updated, then the metadata fallback code can be removed